OWNERS

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
```

Comment:
to confirm, do we want `relasers` or `approvers`?

Reply:
used that to be consistant with charts repo and the link